raiden_libs/contract.py

```diff
@@ -0,0 +1,113 @@
+from typing import List, Any
+
+import rlp
+from eth_utils import decode_hex, encode_hex, denoms
+from ethereum.transactions import Transaction
+from web3 import Web3
+from web3.contract import Contract
+
+from raiden_libs.utils import private_key_to_address, sign_transaction
+
+DEFAULT_TIMEOUT = 60
+DEFAULT_RETRY_INTERVAL = 3
+GAS_PRICE = 20 * denoms.gwei
+GAS_LIMIT_POT = 21000
+GAS_LIMIT_CONTRACT = 130000
+
+
+def create_signed_transaction(
+        private_key: str,
+        web3: Web3,
+        to: str,
```

Comment:
We should add a typing module and make a type for addresses. Like https://github.com/raiden-network/raiden-pathfinding-service/blob/master/pathfinder/utils/types.py

Reply:
easy peasy, but let's make this a separate issue & PR